modules/sr/robot/motor_devices.py

```diff
@@ -46,6 +61,13 @@ def set_speed(self, speed: float) -> None:
             motor.setPosition(motor.getMaxPosition())
         motor.setVelocity(abs(speed))
 
+        # If not using braking, set force to 0.1 when motor speed is 0
+        # Otherwise set max force
+        if(speed == 0 and not self._use_brake):
```

Comment:
```suggestion
        if speed == 0 and not self._use_brake:
```
Same again with brackets here.